src/components/panels/hours-panel.js

```diff
@@ -87,72 +87,74 @@ export function HoursPanelNextPrev({ location }) {
   );
 }
 
-function IconWrapper(props) {
+function IconWrapper (props) {
   return (
     <span
       css={{
         display: 'inline-block',
-        marginTop: '-2px',
+        marginTop: '-2px'
       }}
       {...props}
     />
   );
 }
 
-function PreviousNextWeekButton({ type, children, ...rest }) {
+function PreviousNextWeekButton ({ type, children, ...rest }) {
   return (
-    <React.Fragment>
+    <>
       <Button
         {...rest}
-        kind="subtle"
+        kind='subtle'
         css={{
           display: 'none',
           [MEDIA_QUERIES.LARGESCREEN]: {
-            display: 'flex',
-          },
+            display: 'flex'
+          }
         }}
       >
         {type === 'previous' && (
           <IconWrapper>
             <Icon
-              icon="navigate_before"
+              icon='navigate_before'
               css={{ marginRight: SPACING['2XS'] }}
             />
           </IconWrapper>
         )}
         {children}
         {type === 'next' && (
           <IconWrapper>
-            <Icon icon="navigate_next" css={{ marginLeft: SPACING['2XS'] }} />
+            <Icon icon='navigate_next' css={{ marginLeft: SPACING['2XS'] }} />
           </IconWrapper>
         )}
       </Button>
       <Button
         {...rest}
-        kind="subtle"
+        kind='subtle'
         css={{
           display: 'flex',
           [MEDIA_QUERIES.LARGESCREEN]: {
-            display: 'none',
-          },
+            display: 'none'
+          }
         }}
       >
         <IconWrapper>
-          {type === 'previous' ? (
-            <Icon icon="navigate_before" />
-          ) : (
-            <Icon icon="navigate_next" />
-          )}
+          {type === 'previous'
+            ? (
+              <Icon icon='navigate_before' />
+              )
+            : (
+              <Icon icon='navigate_next' />
+              )}
```

Comment:
You could simplify this to just one line:
```suggestion
          <Icon icon={ type === 'previous' ? 'navigate_before' : 'navigate_next' } />
```

Reply:
Much cleaner! Thanks!